site/content/contribute/server/developer-workflow.md

```diff
@@ -50,6 +50,8 @@ Some useful `make` commands:
 * `make nuke` wipes your local environment back to a completely fresh start.
 * `make package` creates packages for distributing your builds and puts them in the `./dist` directory. First you will need to run `make build` and `make build-client`.
 
+If you would like to run the development environment without Docker you can set the `MM_NO_DOCKER` environment variable.
```

Comment:
Should we elaborate at all on what this means? (i.e. setup your own database instance)

Reply:
Good call. Added.